From 16dd9f42f911def97236f58be72d6d6aa023373e Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Fri, 2 Jul 2021 15:01:54 +0200
Subject: [PATCH 102/N] [squash] CI: use -O2

Without optimizations the stack size increases and tests start to fail
with stack overflow (this is a known problem upstream https://bugs.python.org/issue43271)

Either we enforce -O2 or we increase the stack size.

Needs to be merged into the CI setup
---
 .github/workflows/mingw.yml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.github/workflows/mingw.yml b/.github/workflows/mingw.yml
index fff5fa5..8486bb4 100644
--- a/.github/workflows/mingw.yml
+++ b/.github/workflows/mingw.yml
@@ -52,6 +52,7 @@ jobs:
           set -ex
           pkgver='3.9.5'
           _pybasever='3.9'
+          CFLAGS="-O2"
           
           PREFIX_WIN=$(cygpath -wm ${MINGW_PREFIX})
           if [ ${{ matrix.msystem }} == "CLANG64" ]
-- 
2.32.0

